artifactory/commands/transferconfig/transferconfigmerge_test.go

```diff
@@ -0,0 +1,111 @@
+package transferconfig
+
+import (
+	"github.com/jfrog/jfrog-client-go/access/services"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+const (
+	QuotaNumber = 1073741825
+)
+
+var (
+	tcc TransferConfigCommand
+)
+
```

Comment:
It looks like there are a lot of tests that are missing. For example, there's no tests for the repositories conflicts validation and no tests which checks the CSV outputs.
Let's complete to make sure we're fully covered.

Reply:
We talked about this in zoom